bazel/deps.bzl

```diff
@@ -0,0 +1,89 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
+
+def opencensus_cpp_deps():
+    """Loads dependencies need to compile the opencensus-cpp library."""
+
+    maybe(
+        http_archive,
+        name = "com_google_protobuf",
```

Comment:
Where did this come from? The current `WORKSPACE` doesn't load `com_google_protobuf`.

Reply:
I pasted it here maybe when I try to use maybe function.  😿 
Deleted.